elm-app/src/elm/Translate.elm

```diff
@@ -24,6 +24,7 @@ type TranslationId
     | MatchingResults
     | MoreDetailsText
     | PriceText
+    | LocationText
```

Comment:
@amitaibu I'm not really sure I understand this, do you want me to add the location to it. 
i.e. `english = "Where: " ++ eventLocation`